### PR TITLE
Move icons to ResourceIcons and add price (refs #415)

### LINKED ISSUE #415: Show resource price in resource list items
The price can be a range for example 5 - 50e/h or it can just be one value for example 10e/h.

Resource will have `min_price_per_hour` and `max_price_per_hour` containing the price in strings or a null if price is not set.

See mockups page 9.

### PR DESCRIPTION
**Pull request by YoYuUm**
Closes #415

### CODE REVIEW

File: app/screens/shared/resource-list/ResourceIcons.spec.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    it('renders correct text', () => {
+      expect(spanText.text()).to.equal('2h');
+    });
+
+    it('is not rendered if prop is not passed', () => {
+      expect(wrapperNoProps.find({ glyph: 'time' })).to.have.length(0);
+    });
+  });
+
+  describe('time icon', () => {
```
Comment (lraivio) [inline app/screens/shared/resource-list/ResourceIcons.spec.js]: - [ ] This should be "euro icon" or maybe "price"

File: app/screens/shared/resource-list/ResourceIcons.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    </span>
+  );
+}
+
+function getHourlyPrice({ minPricePerHour, maxPricePerHour }) {
+  if (!(minPricePerHour || maxPricePerHour)) {
+    return null;
+  }
+  if (minPricePerHour && maxPricePerHour) {
+    return `${Number(minPricePerHour)} - ${Number(maxPricePerHour)}e/h`;
```
Comment (lraivio) [inline app/screens/shared/resource-list/ResourceIcons.js]: - [ ] You could also add a check for a case where `minPricePerHour` and `maxPricePerHour` are equal for example both '10.00' so we avoid displaying '10 - 10e/h'. In those cases only show '10e/h'

File: app/screens/shared/resource-list/ResourceIcons.spec.js (hunk 1 above)
Comment (YoYuUm) [inline app/screens/shared/resource-list/ResourceIcons.spec.js]: I ammended the commit with this change.

**Comment by lraivio** [comment]
Nice! Looks good 👍

**Pull request merged**